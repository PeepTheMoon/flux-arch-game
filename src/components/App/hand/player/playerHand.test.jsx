import React from 'react';
import { shallow } from 'enzyme';
import playerHand from './PlayerHand';

describe('playerHand component', () => {
  it('renders playerHand', () => {
    const wrapper = shallow(<playerHand image="https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcRolJqqHMjpTKMX1xJ5ytee8332VT_vLI4PoAsurW4zcQQrE8Nd&usqp=CAU" name='paper'/>);
    expect(wrapper).toMatchSnapshot();
  });
});
