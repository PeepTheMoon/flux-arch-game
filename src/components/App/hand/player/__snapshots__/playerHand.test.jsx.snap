// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`playerHand component renders playerHand 1`] = `
<playerHand
  image="https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcRolJqqHMjpTKMX1xJ5ytee8332VT_vLI4PoAsurW4zcQQrE8Nd&usqp=CAU"
  name="paper"
/>
`;
